src/App.jsx

```diff
@@ -1,5 +1,37 @@
+import { useCookies } from 'react-cookie';
+import { useState, useEffect } from 'react';
+import ApiService from './services/ForecastApiService';
+
 function App() {
-  return <div>Hello world!</div>;
+  const [cookies] = useCookies(['token']);
+  const [locations, setLocations] = useState([]);
+
+  useEffect(async () => {
+    const searchedLocations = await getSearchLocation('London', cookies);
```

Comment:
getLocations? searchLocations? 

const searchedLocations = whateverYouPreferButInPlural. or viceversa

Reply:
Done. Here we have to use plural form because these methods work with location data array.